platform/ios/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 * Fixed a custom geometry source bug caused by using the outdated tiles after style update [#15112](https://github.com/mapbox/mapbox-gl-native/pull/15112)
 
+### Offline maps
+
+* Added methods to invalidate and clear resources in the offline database. [#14978](https://github.com/mapbox/mapbox-gl-native/pull/14978)
```

Comment:
I think this misses the method to resize the ambient cache, and also doesn't make it clear that the methods affect the ambient cache part of the database. Suggestion:

`* Added methods to clear the ambient cache, change the size of the ambient cache, and delete and revalidate the database that contains the ambient and offline cache. These provide greater developer control over the impact of caching. [#14978](https://github.com/mapbox/mapbox-gl-native/pull/14978)`

@tmpsantos @zugaldia @julianrex thoughts?

Reply:
Great - very clear.